server/src/main/java/org/elasticsearch/TransportVersion.java

```diff
@@ -205,7 +206,7 @@ private static TransportVersion registerTransportVersion(int id, String uniqueId
      */
 
     private static class CurrentHolder {
-        private static final TransportVersion CURRENT = findCurrent(V_8_500_068);
+        private static final TransportVersion CURRENT = findCurrent(V_8_500_069);
```

Comment:
@carlosdelest, @kderusso 

A few related questions:

- Do we need to make any more changes related TransportVersion here? 
- If we remove `indices` information from the binary_content and increase the current TransportVersion, will this prevent de(serialize) the content from already existing old index? I find this part challenging to understand. Can you please redirect to code how TransportVersion resolves this issue.

Reply:
@carlosdelest Thank you for the explanation. The [Using Transport Version doc](https://docs.google.com/document/d/1zOUr_MIGOsK44T7Qfv6hrHXEw3rN3_7SzorOXyzrmNk/edit#heading=h.uojxbflxmphu) is very helpful in order to understand how TransportVersion works.